<launch>
	<!-- ========================================================== -->
	<!-- Configuration of the strokelitude                          -->
	<!-- ========================================================== -->


    <param name="bodyangle"                      type="int"      value="0" />
  
    <param name="wing/radius_outer"              type="int"      value="30" />
    <param name="wing/radius_inner"              type="int"      value="10" />
    <param name="wing/angle1"                    type="int"      value="-45" />
    <param name="wing/angle2"                    type="int"      value="45" />
    <param name="wing/nbins"                     type="int"      value="360" />
  
    <param name="wing/right/origin/x"            type="int"      value="0" />
    <param name="wing/right/origin/y"            type="int"      value="0" />
    <param name="wing/right/threshold"           type="double"   value="0.4" />
    <param name="wing/right/flight_threshold"    type="double"   value="0.2" />
  
    <param name="wing/left/origin/x"             type="int"      value="0" />
    <param name="wing/left/origin/y"             type="int"      value="0" />
    <param name="wing/left/threshold"            type="double"   value="0.4" />
    <param name="wing/left/flight_threshold"     type="double"   value="0.2" />

</launch>
